ISICbalance/BalanceViewController.swift

```diff
@@ -23,12 +23,21 @@ class BalanceViewController: UIViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        navigationController?.isNavigationBarHidden = true
         self.balanceView.accountButton?.addTarget(self, action: #selector(accountBtnHandle), for: .touchDown)
     }
     
     @objc func accountBtnHandle() {
         let VC = AccountViewController()
         navigationController?.pushViewController(VC, animated: true)
     }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        navigationController?.isNavigationBarHidden = true
```

Comment:
👍 už jenom drobnost, nenastavuj to přímo přes `isNavigationBarHidden`, ale použij metodu `setNavigationBarHidden(:animated:)` a předej tam `animated` parametr z `viewWillAppear` nebo `viewWillDisappear`.

Reply:
Jo ajo. 🤦‍♂️ Špatně jsem se do toho objc kódu podíval.